docs/php/how-to-develop-an-activity-definition-in-php.md

```diff
@@ -0,0 +1,54 @@
+---
+id: how-to-develop-an-activity-definition-in-php
+title: Develop an Activity Definition in PHP
+sidebar_label: Activity Definition
+---
+
+Activities are defined as methods of a plain PHP interface annotated with `#[ActivityInterface]`.
+(You can also use PHP 8 attributes in PHP 7.)
+
+Each method defines a single Activity type.
+A single Workflow can use more than one Activity interface and call more than one Activity method from the same interface.
+
+The only requirement is that Activity method arguments and return values are serializable to a byte array using the provided [DataConverter](https://github.com/temporalio/sdk-php/blob/master/src/DataConverter/DataConverterInterface.php) interface.
+The default implementation uses a JSON serializer, but an alternative implementation can be easily configured.
+
+Following is an example of an interface that defines four Activities:
+
+```php
+#[ActivityInterface]
+interface FileProcessingActivities
```

Comment:
YourActivityInterface

And let's scale down the initial example to a single method - 
Then do a follow up showing that you can have multiple Activity methods that can use shared class variables and methods.

But keep the method names generic and not use case specific such as "file processing" etc --